src/Magnum/ImGuiIntegration/Test/IntegrationTest.cpp

```diff
@@ -35,19 +35,40 @@ namespace Magnum { namespace ImGuiIntegration { namespace Test { namespace {
 struct IntegrationTest: TestSuite::Tester {
     explicit IntegrationTest();
 
+#ifdef IMGUI_HAS_IMSTR
+    void stringView();
+#endif
     void vector2();
     void vector4();
     void color();
     void colorLiterals();
 };
 
 IntegrationTest::IntegrationTest() {
-    addTests({&IntegrationTest::vector2,
+    addTests({
+#ifdef IMGUI_HAS_IMSTR
+              &IntegrationTest::stringView,
+#endif
+              &IntegrationTest::vector2,
               &IntegrationTest::vector4,
               &IntegrationTest::color,
-              &IntegrationTest::colorLiterals});
+              &IntegrationTest::colorLiterals
+              });
 }
 
+#ifdef IMGUI_HAS_IMSTR
+void IntegrationTest::stringView() {
+    using Containers::StringView;
+    StringView stringView("Hello World!");
+    ImStrv imStrv{"Hello World!"};
+
+    CORRADE_COMPARE(StringView(imStrv), stringView);
+
+    ImStrv c(stringView);
+    CORRADE_COMPARE(c, stringView);
```

Comment:
Hm, as far as I can see, this would convert `c` to a StringView and then use Corrade's own equality comparison. Which I don't think is wanted, since it is no different from the other comparison above.

Instead, isn't there something like `ImStrCmp` that could be used here?